src/c++/perf_analyzer/perf_utils.h

```diff
@@ -36,6 +36,7 @@
 #include <iostream>
 #include <memory>
 #include <random>
+#include "client_backend/client_backend.h"
```

Comment:
Actually, do you still need to preserve the ordering below after you included this file?

Reply:
No. We are using the order as dictated by the formatter.